src/Domain/Metrics/Style/Style.php

```diff
@@ -181,7 +180,6 @@ public function getInsights(): array
             SingleQuoteFixer::class,
             StandardizeNotEqualsFixer::class,
             PhpdocIndentFixer::class,
-            PhpdocInlineTagFixer::class,
```

Comment:
This fixer doesn't exist anymore ? Any replacement ?

Reply:
@Jibbarth Apparently it still exists! But they just renamed it. Added with the new name.